### ISSUE TITLE
Segmentation error when attempting to install MVT on Linux Manjaro

### ISSUE THREAD

**Issue by bkgrrl00** [mvt-project/mvt#374]
I just tried to install MVT on Linux Manjaro via:

`sudo pamac install mvt`

But the installation failed, due to a segmentation error. How can I resolve this?

**Comment by Te-k** [comment]
Hi,

I just discovered that there is [an MVT package](https://software.manjaro.org/package/mvt) for Manjaro, but we do not maintain this package. I would recommend to either reach out to the package maintainer to debug this issue, or install the package from [pip](https://docs.mvt.re/en/latest/install/) instead.

**Comment by bkgrrl00** [comment]
Got it. Will do.

**Comment by bkgrrl00** [comment]
I just tried to follow the Linux installation instructions posted [here](https://docs.mvt.re/en/latest/install/), but the instructions don't work for Arch-based Linux distros, such as Manjaro. So I guess I won't be able to install the latest version of MVT.

**Comment by Te-k** [comment]
You should, you just need to find the archlinux packages equivalent of `python3 python3-pip libusb-1.0-0 sqlite3` (probably [sqlite](https://archlinux.org/packages/core/x86_64/sqlite/) [libusb](https://archlinux.org/packages/core/x86_64/libusb/) python and [python-pip](https://archlinux.org/packages/extra/any/python-pip/). Then install from pip and it should work (please let me know if it does/doesn't, we can update the MVT doc to reflect that)

**Comment by kpcyrd** [comment]
I'm one of the maintainers of the MVT package in Arch Linux, can you please post the log of the specific error you're getting, together with the command you're executing?

If the installation of the Arch Linux package itself failed with a segmentation error this means there's something wrong with your system and/or computer, the MVT application is only extracted to the filesystem during installation (so the segmentation error is not coming from MVT, but from your package manager).

**Comment by bkgrrl00** [comment]
I've since done a fresh install of Manjaro and unfortunately I didn't save the exact error message. If I recall correctly, I got the segmentation error message when I attempted to install MVT via [pamac](https://software.manjaro.org/package/mvt).

I was later able to install MVT via the Manjaro GUI installer. However, I've encountered a couple of issues with the Manjaro GUI version of MVT:

(1) Although the installation was apparently successful, when I attempted to run MVT I got an error message that a dependency was missing. The dependency was something like pyahocorasick. I found a similar-sounding package in the Manjaro GUI installer and, after had I installed it, I was able to get MVT to work. However, when I now search in the Manjaro GUI installer for the similar-sounding package that had previously enabled the MVT installation to work, I can't find it any more.

(2) When I attempt to install MVT on Manjaro, either via pamac or via the GUI package manager, I get prompted to completely update my system. I understand that MVT has some dependencies, but a complete system upgrade seems excessive. I have a policy of disabling automatic updates, as I am constantly battling spyware, so I'm not comfortable with such an extensive upgrade. Is it really necessary?

I apologize for not being more specific, but because installing MVT on Manjaro requires a complete system upgrade (at least, if I avoid the pip route), I don't feel comfortable installing it at the moment, which I would have to do in order to determine the exact error message.
